oauthenticator/github.py

```diff
@@ -154,6 +154,8 @@ async def authenticate(self, handler, data=None):
         else:
             raise web.HTTPError(500, "Bad response: {}".format(resp_json))
 
+        granted_scopes = resp_json['scope'].split(',')
```

Comment:
This is probably fine no matter what. I'm not confident we should count on this field being part of the response.

```suggestion
        granted_scopes = []
        if resp_json.get("scope"):
            granted_scopes = resp_json["scope"].split(",")
```

Reply:
Test failure relates to mocked responses I think.

Reply:
yes, i could not find any github docs on the returned response. only pointers that suggest "scope" is in there.